test_models.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+import requests_mock
+import xml.etree.ElementTree as ET
+
+import models
+
+
+@pytest.fixture
+def item():
+    item = ET.fromstring('<records><record><metadata><atom:entry xmlns:atom="http://www.w3.org/2005/Atom"><atom:link rel="alternate" href="http://hdl.handle.net/1234"/><atom:title>Test Title</atom:title><atom:link rel="http://www.openarchives.org/ore/terms/aggregates" href="http://dome.mit.edu/bitstream/handle/1721.3/74331/MC0356_114633.pdf?sequence=1" title="MC0356_114633.pdf" type="application/pdf" length="706590"/><atom:link rel="http://www.openarchives.org/ore/terms/aggregates" href="http://dome.mit.edu/bitstream/handle/1721.3/74331/license.txt?sequence=2" title="license.txt" type="text/plain" length="1748"/></atom:entry></metadata></record></records>')
+
+    return item
+
+
+@pytest.fixture
+def NS():
```

Comment:
Just import the `NS` constant directly from the module. There's no need for the fixture here and you'll also be better off testing the actual namespace dictionary used in the production code.